core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2092,15 +2215,17 @@ class SparkContext(config: SparkConf) extends Logging {
    * @param checkSerializable whether or not to immediately check <tt>f</tt> for serializability
    * @throws SparkException if <tt>checkSerializable</tt> is set but <tt>f</tt> is not
    *   serializable
+   * @return reference cleaned from closures
```

Comment:
Might just say `@return the cleaned closure`; "cleaned from closures" doesn't really scan